spec/API_specification/dataframe_api/__init__.py

```diff
@@ -12,3 +13,27 @@
 String representing the version of the DataFrame API specification to which the
 conforming implementation adheres.
 """
+
+def concat(dataframes: Sequence[DataFrame]) -> DataFrame:
+    """
+    Concatenate DataFrames vertically.
+
+    To concatenate horizontally, please use ``insert``.
+
+    Parameters
+    ----------
+    dataframes : Sequence[DataFrame]
+        DataFrames to concatenate.
+        Column names, ordering, and dtypes must match.
+
+    Notes
+    -----
+    The order in which the input DataFrames appear in
+    the output is preserved (so long as the DataFrame implementation supports ordered
+    columns).
```

Comment:
"ordered columns" is unclear to me where I interpreted it as having independent orderings which isn't really the case. How about using "row ordering" instead:
```suggestion
    the output is preserved (so long as the DataFrame implementation supports row
    ordering).
```